compiler/one-cmds/dummy-driver/CMakeLists.txt

```diff
@@ -1,11 +1,18 @@
 # dummy driver for interface test
-set(DUMMY_DRIVER_SRC src/dummy-compile.cpp)
```

Comment:
can you please leave this as is and add same kind for `help-compile`?

Reply:
@seanshpark You mean this way?
```
set(HELP_DRIVER_SRC src/help-compile.cpp)
```

Reply:
@seanshpark I thought that way  but `DUMMY_DRIVER_SRC` means dummy driver source. And `help-compile` is also dummy driver.. If it seems okay, I'll update as you said.

Reply:
This folder is dummy-driver so I think you are adding help-compile to dummy-driver is somewhat NOT what you may want...

1. rename dummy-driver to something else that can hold both executables
2. add help-compile to another driver

Reply:
> rename dummy-driver to something else that can hold both executables

Well, I think the name `dummy-driver` already can holds both executables because `dummy-compile` and `help-compile` are dummies.

> add help-compile to another driver

I don't understand the point. Maybe you asked me to create a new folder whose name is `help-driver` or sth and then add `help-compile` to there?

Reply:
> Well, I think the name dummy-driver already can holds both executables because dummy-compile and help-compile are dummies.

OK, let's go with this

> I don't understand the point.

My mistake... forget about second one...